src/app/hooks/useProductCategory.ts

```diff
@@ -1,7 +1,7 @@
 'use client'
 import useSWR from 'swr';
 import getData from '@app/helpers/getData';
-import { HomeCategoryProductsData, ProductCategoryData } from '@app/types';
+import { HomeCategoryProductsData, ProductCategoryData, TProductCategory } from '@app/types';
```

Comment:
Similar to this https://github.com/Kasia-Africa/kasia-frontend/pull/1/files#r1485210910, remove `TProductCategory `types as well